.github/workflows/tests.yml

```diff
@@ -20,7 +20,8 @@ jobs:
       - name: Run pylint
         run: |
           cd osbuild
-          pylint osbuild runners/* assemblers/* stages/* sources/*
+          find . -type f -name "*.py" | xargs pylint
+          pylint runners/* assemblers/* stages/* sources/*
```

Comment:
The second line is not necessary, is it?

Reply:
It is, we do not use `.py` suffixes for those. I thought about whether we can simplify it, but then decided that it is ok this way. These directories are kinda API, so I was fine specifying them directly and not using a `.py` suffix there. At least that would be a separate discussion, I guess.

Reply:
Of course. Didn't think about the missing extension. Thanks!